### ISSUE TITLE
Fine-tuning ShapeHD always generates same weird result

### ISSUE THREAD

**Issue by rozentill** [xiumingzhang/GenRe-ShapeHD#19]
Hi,

I am fine-tuning shapeHD with a pre-trained 3D GAN, but it seems all the test model will lead to this result.

![Capture](https://user-images.githubusercontent.com/11246898/59996134-59188b00-960e-11e9-9f14-d5b0284d5625.PNG)

Any idea about this?

Thank you!

**Comment by ztzhang** [comment]
this might be an artifact due to bad weight balancing between Gan loss and cross entropy loss. The way we balance those two losses are looking at corresponding gradient variance. Maybe @xiuming can share more insights into this.

**Comment by xiumingzhang** [comment]
Is this a visualization issue? Looks to me the voxels to the cube sides are all occupied. Could be that your threshold is too low, or you took log when you shouldn't, or you didn't take log when you should?

**Comment by rozentill** [comment]
Thanks, I changed the weight to 1e-5 and works now.
